packages/web-components/fast-components/docs/color.md

```diff
@@ -0,0 +1,143 @@
+| `neutralLayerL1` | Used as the background color for the primary content layer (L1). |
+| `neutralLayerL1Alt` | Alternate color for L1 surfaces. |
+| `neutralLayerL2` | Used as the background for the top command surface, logically below L1. |
+| `neutralLayerL3` | Used as the background for secondary command surfaces, logically below L2. |
+| `neutralLayerL4` | Used as the background for the lowest command surface or title bar, logically below L3. |
+
+### Neutral Recipes
+#### Text
+Neutral text recipes address *most* cases of text used in a UI, from interactive element text, headings, and body text.
+
+| name | Description |
+|------|-------------|
+| `neutralForegroundRest` | Primary page text color when the text is in a *rest* state. |
+| `neutralForegroundHover` | Primary page text color when the text is in a *hover* state. |
+| `neutralForegroundActive` | Primary page text color when the text is in a *active* (pressed) state. |
+| `neutralForegroundHint` | Secondary *hinting* text to be used with [non-large text](https://www.w3.org/TR/WCAG/#contrast-minimum) to meet a 4.5:1 contrast ratio to the background. |
+| `neutralForegroundHintLarge` | Secondary *hinting* text to be used with [large text](https://www.w3.org/TR/WCAG/#contrast-minimum) to meet a 3:1 contrast ratio to the background. |
+
+#### Fills (Backgrounds)
+Neutral fills are indented to be used as fill colors (background) to UI elements to distinguish them from the background. 
+
+| name | Description |
+|------|-------------|
+| `neutralFillRest`| Used to visually differentiate the UI element from it's background context at rest. |
+| `neutralFillHover`| Used as the fill of a `neutralFill` element when hovered. |
+| `neutralFillActive`| Used as the fill of a `neutralFill` element when active. |
+| `neutralFillSelected`| Used as the fill of a `neutralFill` element when selected. |
+| `neutralFillStealthRest`| Used when a UI element's fill is visually differentiated when being interacted with by the user. |
+| `neutralFillStealthHover`| Used as the fill of a `neutralFillStealth` element when hovered. |
+| `neutralFillStealthActive`| Used as the fill of a `neutralFillStealth` element when active. |
+| `neutralFillStealthSelected`| Used as the fill of a `neutralFillStealth` element when selected. |
+
+#### Outlines and Dividers
+Neutral outlines are used to construct outline controls and dividers.
+
+| name | Description |
+|------|-------------|
+| `neutralOutlineRest` | Used as a rest outline color for outline controls. |
+| `neutralOutlineHover` | Used as the outline of a `neutralOutline` control when hovered. |
+| `neutralOutlineActive` | Used as the outline of a `neutralOutline` control when active.  |
+| `neutralDividerRest` | Used as the color for divider elements. |
+
+#### Toggles
+Toggles have a set of recipes that are specifically tailored.
```

Comment:
Can we change this sentence a little, maybe "Toggles have a specific set of recipes."